src/journal/logic.py

```diff
@@ -505,6 +505,8 @@ def resend_email(article, log_entry, request, form):
 
 
 def send_email(user, form, request, article):
+    cc_list = [x.strip() for x in form.cleaned_data['cc'].split(';') if x]
+    valid_cc_email_addresses = validate_to_list(cc_list)
```

Comment:
This function can raise a ValidationError, which would hit the top level and throw a 500
We should do the validation in the form instead.

Also: if you follow my recommendation above about using tagit, we could have front-end validation too.